resources/js/ziggy.d.ts

```diff
@@ -309,6 +309,27 @@ declare module 'ziggy-js' {
             "required": true
         }
     ],
+    "achievement-set-claim.create": [
```

Comment:
was this required in my PR? or is it bleeding in from the management console PR?

Reply:
It's just picking up the new routes from me needing to regenerate types with `composer types`. I don't think it's a big deal to re-run `composer types` unless something in them changes. Maybe we should explore automating this somehow so it's always current?